### DISCUSSION TITLE
Accessibility: Add focus option for link prefetching

### DISCUSSION THREAD

**Discussion by Jussi-Mannisto** [inertiajs/inertia#2047] [Ideas]
Hi,

Thanks for the great feature!

There's currently no 'focus' strategy for link prefetching, which would be a useful option for keyboard navigation.

I've created a custom component that handles this programmatically, but I'd prefer to use Inertia's link directly.

BR,
Jussi

**Comment by ncphillips** [comment]
@Jussi-Mannisto what does your component look like?
